spec/features/consumer/cookies_spec.rb

```diff
@@ -94,4 +101,29 @@ def expect_not_visible_cookies_banner
   def accept_cookies_button_text
     I18n.t('legal.cookies_banner.cookies_accept_button')
   end
+
+  def visit_root_path_and_wait
+    visit root_path
+    sleep 1
```

Comment:
I'm not very fluent with feature specs but I recall there's a way to tell capybara to wait for a DOM element to show up. These sleeps are scary! they add up a lot of time to the test suite.

Reply:
I know. We tried everything to make Capybara wait here and it didn't work. I don't like the solution either, but after hours of trying different things, this was the only thing that worked. @luisramos0 spent hours as well.

Part of the problem is the concurrent Javascript initialisation after the page loaded. We don't have a trigger for *all callbacks loaded, all triggers executed, everything initialised*. But even with the wait in the beginning, once we click on a button, Capybara's wait is still not reliable.

My metric for a stable spec was to run the same spec 100 times on Travis (25 times on each box).

Reply:
yes, I have spent too much time here... it must be related to the loading of the angular directives...
Pau, if you mean wait_until, it's gone from capybara.

Reply:
🙈 yes, I think it was `wait_until`. Really bad news but... 👍

Reply:
yeah :-/

Reply:
yes we have! :-O